nengo/utils/graphs.py

```diff
@@ -98,6 +98,43 @@ def toposort(edges):
     return ordered
 
 
+def transitive_closure(edges, topo_sorted=None):
+    """Constructs the transitive_closure of a DAG.
+
+    The complexity is O(nodes + vertices).
+
+    Parameters
+    ----------
+    edges : dict
+        Dict of the form {a: {b, c}} where b and c depend on a. Must not
+        contain cycles.
+    topo_sorted : sequence, optional
+        The topological sorting of the vertices. If not passed in the algorithm
+        will do a topological sort.
+
+    Returns
+    -------
+    The transitive closure data structure as `edges`.
```

Comment:
I think the sentence is missing a word. It should say something like it is using the same “data structure as `edges`”. Would that clarify it? Or is it better to repeat the description of the datat structure?